### PR TITLE
Bug: Collection View Content Size Issue #33 is fixed.

### PR DESCRIPTION
**Pull request by gungorbasa**
Bug: Collection View Content Size Issue #33 is fixed.

### CODE REVIEW

File: Sources/MessagesViewController.swift [hunk 1]
```diff
@@ -222,37 +222,40 @@ extension MessagesViewController: UICollectionViewDataSource {
...
+        let height = self.view.frame.size.height
+        let messageInputBarHeight = inputAccessoryView?.bounds.size.height ?? 0
+        let keyboardHeight = keyboardFrame.height - messageInputBarHeight
+        
+        if (keyboard.origin.y + keyboard.size.height) > height {
+            // Hardware keyboard is found
+            messagesCollectionView.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
+            
+        } else {
+            // Software keyboard is found
```
Comment (MacMeDan) [inline Sources/MessagesViewController.swift:252]: I really appreciate these comments. They make understanding what is happening really easy. 💯

**Comment by MacMeDan** [comment]
@SD10 I reviewed this code and ran it on my local machine. It all looks good so I'm going to go ahead and merge it. Great work @gungorbasa 🎉

**Comment by gungorbasa** [comment]
Thank you. Sounds good :)

**Comment by SD10** [comment]
@gungorbasa @MacMeDan Sorry it took me a long time to get around to reviewing this PR. It does solve the problem but by moving it elsewhere. This won't be a straightforward thing to solve due to the ordering of keyboard notifications by the NotificationCenter. I'm probably going to write a custom keyboard tracker. Regardless, thank you for your hard work 💯

**Comment by MacMeDan** [comment]
I think that is fine and since this move us forward in reguards to the current state of the app I like that we have merged it in. Lets definitely make a task for that "Custom keyboard tracker" - that will be super useful in other places as well.

**Pull request merged**